GameLamp/src/Core/EntryPoint.h

```diff
@@ -1,14 +1,17 @@
 #pragma once 
 #include "Core/Application.h"
 #include "Core/Core.h"
-
+#include "Log.h"
```

Comment:
Use full path "Core/Log.h" to prevent errors after moving file to a different location